### PR TITLE
Add `ANALYZE` to Edgedb commands.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Add a plain `ANALYZE` command without options or arguments.

Issue #4938

### CODE REVIEW

**Comment by vpetrovykh** [comment]
I suspect that there might be more `dbstate` stuff that I should account for in my `MaintenanceQuery`.

File: edb/server/compiler/compiler.py [hunk 1]
```diff
@@ -1796,6 +1814,11 @@ def _compile_dispatch_ql(
             caps |= enums.Capability.MODIFICATIONS
         return (query, caps)
 
+    elif isinstance(ql, qlast.AnalyzeStmt):
+        query = _compile_ql_analyze(ctx, ql, script_info=script_info)
+        caps = enums.Capability(0)
```
Comment (msullivan) [inline edb/server/compiler/compiler.py:1819]: We'll probably want some capabilities required here if we ever take this out of dev mode.
Not sure if it matters now, though.

**Review by msullivan — APPROVED** [review]
Looks good!

Could add a non dev mode test (in userddl or some such) for the error message, though it's pretty marginal

File: tests/test_edgeql_explain.py [hunk 2]
```diff
@@ -55,60 +53,11 @@ class TestEdgeQLExplain(tb.QueryTestCase):
             update User set {
               todo += (select .owned_issues filter <int64>.number % 3 = 0)
             };
+
+            analyze;
         '''
     ]
 
-    @classmethod
```
Comment (msullivan) [inline tests/test_edgeql_explain.py:61]: Very happy to have all this gone!

File: edb/edgeql/parser/__init__.py [hunk 3]
```diff
@@ -58,14 +58,37 @@ def parse_fragment(
...
+    parser = qlparser.EdgeQLSingleParser()
+    res = parser.parse(source, filename=filename)
+    assert isinstance(res, qlast.Expr)
+    return res
+
+
 def parse(
     source: Union[qltokenizer.Source, str],
     module_aliases: Optional[Mapping[Optional[str], str]] = None,
+    as_expr: bool = True,
```
Comment (elprans) [inline edb/edgeql/parser/__init__.py:76]: I'd prefer if we had a separate `parse_expr` function instead.

Comment (elprans) [reply edb/edgeql/parser/__init__.py:76]: And maybe then rename `parse` to `parse_command`

**Review by msullivan — APPROVED** [review]
Great

**Pull request merged**